src/jit/compiler.h

```diff
@@ -1933,9 +1933,10 @@ class Compiler
     GenTreePtr gtNewIndexRef(var_types typ, GenTreePtr arrayOp, GenTreePtr indexOp);
 
     GenTreeArgList* gtNewArgList(GenTreePtr op);
-
     GenTreeArgList* gtNewArgList(GenTreePtr op1, GenTreePtr op2);
 
+    GenTreeArgList* gtNewAggregate(GenTree* element);
```

Comment:
I don't think Aggregate is a good term to use here because it is not used anywhere else in the source base. I think that something like gtNewMultiRegStruct would be better.